src/renault_api/const.py

```diff
@@ -0,0 +1,182 @@
+"""Constants for Renault API."""
+CONF_GIGYA_APIKEY = "gigya-api-key"
+CONF_GIGYA_URL = "gigya-api-url"
+CONF_KAMEREON_APIKEY = "kamereon-api-key"
+CONF_KAMEREON_URL = "kamereon-api-url"
+
+AVAILABLE_LOCALES = {
+    "bg_BG": {
+        CONF_GIGYA_URL: "https://accounts.eu1.gigya.com",
+        CONF_GIGYA_APIKEY: "3__3ER_6lFvXEXHTP_faLtq6eEdbKDXd9F5GoKwzRyZq37ZQ-db7mXcLzR1Jtls5sn",  # noqa
+        CONF_KAMEREON_URL: "https://api-wired-prod-1-euw1.wrd-aws.com",
+        CONF_KAMEREON_APIKEY: "oF09WnKqvBDcrQzcW1rJNpjIuy7KdGaB",
```

Comment:
Every `CONF_KAMEREON_APIKEY` are the same, should they change ?

During this time maybe a `KAMEREON_APIKEY` const.
What do you think ?